C/ds/EvaluationOfPostfixExpression.c

```diff
@@ -0,0 +1,77 @@
+/*
+A postfix expression is a collection of operators and operands in which the operator is placed after the operands. 
+To evaluate a postfix expression using Stack data structure we can use the following steps...
+1.Create an empty stack and start scanning the postfix expression from left to right. 
+2.If the element is an operand, push it into the stack.
+3.If the element is an operator O, pop twice and get A and B respectively. Calculate B operator A and push it back to the stack.
+4.When the expression is ended, the value in the stack is the final answer.
+*/
+
+#include <stdio.h>
+#include <unistd.h>
+struct stack
+{
+    int top;
+    float a[50];
+}
+s;
+int main()
+{
+    char ch[50];
```

Comment:
Use a meaningful name for variables. i.e -  postfix[30]  etc. Do the same with variables x, y, z.